setup.cfg

```diff
@@ -47,7 +47,7 @@ install_requires =
     astroid>=2.8.0,<2.9 # (You should also upgrade requirements_test_min.txt)
     isort>=4.2.5,<6
     mccabe>=0.6,<0.7
-    toml>=0.7.1
+    toml>=0.9.2,<0.10
```

Comment:
Just wondering: Why limit it to `<0.10`? The import should work for `>=0.10` as well.

Reply:
As of `0.10` the `TomlDecodeError` was moved from `toml` to `toml.decoder`.

Reply:
See https://github.com/uiri/toml/blob/0.9.2/toml.py vs https://github.com/uiri/toml/blob/0.10.0/toml/decoder.py

Reply:
It did, but the import is still possible
https://github.com/uiri/toml/blob/4935f616ef78c35a968b2473e806d7049eba9af1/toml/__init__.py#L15

Reply:
Ah! I missed that part! In that case, we don't need to specify an upper version indeed.

Reply:
Fixed in latest version.